.cascade-code/Chapman.edu/_components/left_nav/left_nav.vtl

```diff
@@ -50,12 +61,13 @@
             #buildSiblingPageListItem($sibling)
 
         ## Sibling is a folder
-        #elseif ( $sibling.assetType == 'folder' && $isVisible )
+        #elseif ( $sibling.assetType == 'folder' && $isVisible && $sibling.name != '_files' )
             #set ( $indexPage = $_.locatePage("${sibling.path}/index") )
             #set ( $visibleIndex = $indexPage.metadata.getDynamicField('Hide from navigation').value != "Yes" )
             
             ## Only if the index page is also visible
-            #if ( $visibleIndex )
+            ## Also make sure the correct index page was found
+            #if ( $visibleIndex && ( $indexPage.path == "${sibling.path}/index" ) )
```

Comment:
These last two commits are to deal with folders that don't have index pages for whatever reason. They cause problems similar to what we saw in issue #262 

This is the most effective way I think to do it since velocity doesn't return anything helpful like NULL when it doesn't locate something, it leaves it as the previous value of that variable.